src/ribasim_nl/ribasim_nl/discrete_control.py

```diff
@@ -0,0 +1,47 @@
+# %%
+from pandas import DataFrame
+
+
+def control_state(name: str, length: int) -> list[str]:
+    return [f"{name}_{i+1}" for i in range(length)]
+
+
+def condition(
+    values: list[float],
+    node_id: int,
+    listen_feature_id: int,
+    variable: str = "flow_rate",
+    name: str | None = None,
+) -> DataFrame:
+    df = DataFrame({"greater_than": values})
+    df.loc[:, ["node_id"]] = node_id
+    df.loc[:, ["listen_feature_id"]] = listen_feature_id
+    df.loc[:, ["variable"]] = variable
+    df.loc[:, ["remarks"]] = control_state(name, len(df))
```

Comment:
The remarks column in Ribasim Python was needed as a hack, and it now removed. You could still add `meta_remarks` or another better column name.

We plan to stop automatically adding `meta_` prefixes soon.

Reply:
Check, I think labeling the control-condition is typically meta-data, so makes sense.

+1 for stopping automatically adding `meta_` as I see my models get unintentionally contaminated with unwanted columns :-)